backup/test_backup.py

```diff
@@ -49,13 +49,17 @@ def test_start_no_init(node_factory, directory):
         options=opts, cleandir=False, may_fail=True, start=False
     )
 
-    with pytest.raises(ValueError):
```

Comment:
See https://github.com/lightningd/plugins/pull/416/commits/5892bdc13d27e4dceed41795bdae22845232be90 on how I solved that. 

The test still works correctly, we just need to wait for the 30secs timeout. Maybe we can change the test by setting the `wait_for` `TIMEOUT` value specically lower for this test.

Reply:
This commit is part of the PR https://github.com/lightningd/plugins/pull/416

Reply:
Merged that PR, now we need to rebase here :-)